cogs/statistics.py

```diff
@@ -12,6 +14,137 @@ def __init__(self, bot):
         self.counter = Counter()
         self.db = self.bot.database.db.statistics
 
+    @commands.group(aliases=["stats"])
+    async def statistics(self, ctx):
+        """Statistic related commands"""
+        if ctx.invoked_subcommand is None:
+            await self.bot.send_cmd_help(ctx)
+
+    @statistics.command(name="user")
+    async def statistics_user(self, ctx):
+        """Statistics of the user"""
+        await ctx.trigger_typing()
+        cursor = self.db.commands.find({"author": ctx.author.id})
+        data = discord.Embed(
+            description="Command statistics of {0}".format(ctx.author))
+        data = await self.generate_embed(ctx, data, cursor, False)
+        try:
+            await ctx.send(embed=data)
+        except discord.Forbidden:
+            await ctx.send("Need permission to embed links")
+
+    @statistics.command(name="guild")
+    async def statistics_guild(self, ctx):
+        """Statistics of this guild
+
+        Only available on Discord Server"""
```

Comment:
Add a 
```py
@commands.guild_only()
```
decorator and you can get rid of this bit, and the extended help command.

Reply:
Added decorator, removed check for guild_id